sharpy/combat/default_micro_methods.py

```diff
@@ -169,13 +173,25 @@ def focus_fire(
         shoot_air = step.unit_values.can_shoot_air(unit)
         shoot_ground = step.unit_values.can_shoot_ground(unit)
 
-        air_range = step.unit_values.air_range(unit)
-        ground_range = step.unit_values.ground_range(unit)
-        lookup = min(air_range + 3, ground_range + 3)
+        lookup = step.min_range(unit) + 3
```

Comment:
This was `min` in the original code, so I kept that, but I'm wondering if it shouldn't be `max` instead?